### PR TITLE
Include link to BEIPA

### PR DESCRIPTION
**Pull request by ahpook**
There's some in-depth writing and background from @mlinksva and the policy itself is of interest to our OSPO audience, seems good to include here.

### CODE REVIEW

**Review by zkoppert — APPROVED** [review]
🎉 Great addition! Thanks!!

**Comment by ahpook** [comment]
(BTW i couldn't write to the repo so i had to fork it to my own namespace)

**Comment by zkoppert** [comment]
Just escalated permissions.

File: docs/ospo-tools-and-external-guides.md [hunk 1]
```diff
@@ -15,6 +15,10 @@ These are tools published by employees that can be used by OSPOs for managing op
 - [GitHub Action: Allow users to configure their risk threshold for security issues reported by GitHub Code Scanning, Secret Scanning and Dependabot Security.](https://github.com/marketplace/actions/ghas-policy-as-code)
 - [Safe Settings - an app to manage policy-as-code and apply repository settings to repositories across an organization.](https://github.com/github/safe-settings)
 
+### Policy
+
+- [GitHub's Balanced Employee IP Agreement](https://github.com/github/balanced-employee-ip-agreement) - A policy which balances preserving the ability of employees to work on open source projects with the need to protect GitHub's intellectual property.
+
 ### User administration
```
Comment (Mixie71) [inline docs/ospo-tools-and-external-guides.md:22]: butak1971@gmail.com

Comment (Mixie71) [reply docs/ospo-tools-and-external-guides.md:22]: butak1971@gmail.com

**Review by Mixie71 — COMMENTED** [review]
butak1971@gmail.com

**Pull request merged**